django/db/migrations/migration.py

```diff
@@ -162,8 +162,10 @@ def unapply(self, project_state, schema_editor, collect_sql=False):
                 schema_editor.collected_sql.append("--")
                 if not operation.reduces_to_sql:
                     continue
-            if not schema_editor.connection.features.can_rollback_ddl and operation.atomic:
-                # We're forcing a transaction on a non-transactional-DDL backend
+            atomic_operation = operation.atomic or (self.atomic and operation.atomic is not False)
```

Comment:
@timgraham (or anyone else): I'm having trouble getting my test environment set up to run the test suite against a postgres environment.  Can you point me to anything that might help me figure out what's wrong?

If it helps identify something obviously wrong, my current setup has postgres running in a docker container and runnings tests with `./runtests.py --settings=test_postgres`.  My `test_postgres.py` contents:

```python
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'postgres',
        'USER': 'postgres',
        'HOST': 'localhost',
        'PORT': 5432,
    }
}
SECRET_KEY = "django_tests_secret_key"

# Use a fast hasher to speed up tests.
PASSWORD_HASHERS = [
    'django.contrib.auth.hashers.MD5PasswordHasher',
]
```

I'd expect this to work, but `runtests` results in:
```
Ran 11582 tests in 179.446s

FAILED (failures=7, errors=68, skipped=282, expected failures=8)
```

Anything obviously wrong here?